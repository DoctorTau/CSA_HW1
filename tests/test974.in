144
4846 -3831 -1606 4878 -7417 -9307 6408 -8620 -7162 -5026 -2819 1385 -3022 2719 9502 -4962 624 2526 9381 -6442 6498 6699 -6097 -9115 3362 -443 7475 -225 -3059 -2346 6193 -6157 -4136 1500 6583 -1541 5897 -5923 4452 8105 -2896 8986 2905 8050 7091 -4524 141 776 8749 8002 6078 -920 -6014 -9997 -1725 -7012 -9802 -3051 8027 7348 -8113 -1351 8477 -413 3710 -673 5107 1133 5428 -9025 -7702 -8620 5765 -3672 -1486 -8051 4895 -8584 -3364 4364 6072 -2143 -4780 -1864 -3379 -5415 -9200 7420 1028 3302 -1348 -2664 -3430 8832 2844 7605 -3457 -5177 5174 -429 -3870 -6677 -8131 3839 2114 7380 -6802 328 -6829 4523 -4181 -4899 -5408 5167 -6506 -9251 9573 -6514 -8297 -1647 -3473 8773 1451 5273 91 3634 972 -3745 4373 5353 -1608 -5477 -3131 1209 659 2917 -5642 -3604 -4532 2218 6802 5141 -5737 -8095 