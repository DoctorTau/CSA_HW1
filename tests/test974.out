-1606 -8620 -7162 -5026 -3022 -4962 -6442 -2346 -4136 -2896 -4524 -920 -6014 -7012 -9802 -7702 -8620 -3672 -1486 -8584 -3364 -4780 -1864 -9200 -1348 -2664 -3430 -3870 -6802 -5408 -6506 -6514 -1608 -5642 -3604 -4532 